src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -46,4 +62,32 @@ abstract class IntegrationTestBase {
     actionPlanRepository.deleteAll() // Will also remove all Goals and Steps due to cascade
     timelineRepository.deleteAll() // Will also remove all TimelineEvents due to cascade
   }
+
+  fun getActionPlan(prisonNumber: String): ActionPlanResponse =
```

Comment:
I've added these convenience functions to the integration test superclass, so that all integration tests can use these to setup and retrieve data via the REST APIs (at the moment its mostly `CreateGoalsIntegrationTest` that use these functions, but we can refactor the other tests over time)

Reply:
Nice. I nearly did the same the other day 👍